airflow/models/crypto.py

```diff
@@ -58,7 +57,7 @@ def encrypt(self, b):
         return b
 
 
-_fernet = None  # type: Optional[FernetProtocol]
+_fernet = None  # type: FernetProtocol | None
```

Comment:
Should we replace it with ACTUAL typy hint, rather than comment ? (there are few other similar cases)

Reply:
I believe yes, would be much cleaner indeed ! Thanks

Reply:
> If you can actually check it in Python 3.7 IDE to see if autocomplete works for one of those, we should be good :)

#### Seems fine in neovim (pyright + mypy + flake):
![image](https://user-images.githubusercontent.com/14861206/193948632-a529a5d4-7557-4a7a-b026-212f8c1f70e0.png)
![image](https://user-images.githubusercontent.com/14861206/193948509-3486f431-1621-4153-9b06-b64b14edfbcc.png)


#### Well, it hurts me a bit to launch my dusty Pycharm, but after 10 minutes of painfull indexing :joy:, it looks ok:
![image](https://user-images.githubusercontent.com/14861206/193948271-71603dfe-af6a-4712-821d-d2772c704e8b.png)
![image](https://user-images.githubusercontent.com/14861206/193948299-39a14b69-c23e-4033-8669-90dcf0338e8d.png)
![image](https://user-images.githubusercontent.com/14861206/193948337-8a26c06a-8496-489b-a61c-a04c8e745eb2.png)

Reply:
The key is to launch it more often :). And it does help at times to be more productive even if I fall back to vim  whenever I want to do something fast.